packages/cashier/src/Sass/account-transfer-form.scss

```diff
@@ -0,0 +1,175 @@
+.account-transfer-form {
+    &__hint {
+        color: var(--text-general);
+    }
+    &__submit-button {
+        @include desktop {
+            margin: 3.4rem 0 2.4rem;
+        }
+        @include mobile {
+            margin: 2.4rem 0;
+        }
+    }
+    &__crypto {
+        &--percentage-selector {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            margin-bottom: 2.4rem;
+
+            & .dc-text {
+                margin-top: 0.4rem;
+            }
```

Comment:
The `Text` component has the `className` prop where you can give a class for this. Please try not to change `components` package classes as much as possible.